packages/core/src/codewhisperer/commands/basicCommands.ts

```diff
@@ -371,11 +371,23 @@ const registerToolkitApiCallbackOnce = once(async () => {
             async (connection: AwsConnection) => {
                 getLogger().info(`toolkitApi: connection change callback ${connection.id}`)
                 await AuthUtil.instance.onUpdateConnection(connection)
+                await vscode.commands.executeCommand(
+                    'aws.amazonq.refresh',
+                    (
+                        await getChatAuthState()
```

Comment:
Q always pushes connection status to toolkit using `setConnection`, therefore, without this command, toolkit still knows the status of Q connection.

Reply:
I am trying it locally without this change and it is not updating for me